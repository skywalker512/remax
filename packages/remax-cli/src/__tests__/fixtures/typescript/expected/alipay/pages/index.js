'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

var _rollupPluginBabelHelpers = require('../_virtual/_rollupPluginBabelHelpers.js');
var React = require('react');
var createPageConfig = require('../npm/remax/esm/createPageConfig.js');
require('../npm/remax/esm/index.js');
var View = require('../components/View.js');
var Text = require('../components/Text.js');

var _page = function _page() {
  var _, _obj$a;

  var props = {};
  var TextElement = React.cloneElement(React.createElement(Text.default, null));

  var handleClick = function handleClick() {
    return void 0;
  };

  var handleTouchStart = function handleTouchStart() {
    return void 0;
  };

  var obj = {};
  var value = (_ = 0) !== null && _ !== void 0 ? _ : 1;
  return React.createElement(View.default, null, React.createElement(View.default, _rollupPluginBabelHelpers.extends({
    onClick: handleClick,
    onTouchStart: handleTouchStart,
    id: "view"
  }, props), "foo", obj === null || obj === void 0 ? void 0 : (_obj$a = obj.a) === null || _obj$a === void 0 ? void 0 : _obj$a.b, value), TextElement);
};

var index = Page(createPageConfig.default(_page));

exports.default = index;
