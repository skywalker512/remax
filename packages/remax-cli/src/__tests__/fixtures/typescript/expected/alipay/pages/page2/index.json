{
  "defaultTitle": "Alipay Page",
  "titleBarColor": "#323239",
  "usingComponents": {}
}